vep/__init__.py

```diff
@@ -159,7 +159,7 @@ def symlink_entry_points(self):
         mkdir = sh.Command('mkdir')
         mkdir('-p', "%s/bin" % self.build_dir)
         rcp = RawConfigParser()
-        # someone could be foolish enough to use a hypen in ther package name, needs to be a _.
+        # someone could be foolish enough to use a hypen in their package name, needs to be a _.
```

Comment:
"hypen" should probably be "hyphen", if we're going to correct the comment's English. :)